zwave_js_server/model/node.py

```diff
@@ -104,57 +104,57 @@ def ready(self) -> Optional[bool]:
     @property
     def device_class(self) -> DeviceClass:
         """Return the device_class."""
-        return DeviceClass(self.data["deviceClass"])
+        return DeviceClass(self.data.get("deviceClass", {}))
 
     @property
-    def is_listening(self) -> bool:
+    def is_listening(self) -> Optional[bool]:
         """Return the is_listening."""
         return self.data["isListening"]
```

Comment:
If the item is optional we need to use dict.get.

Reply:
Oh I forgot one ;-)